PySpice/Spice/HSpice/hspicefile/hspice_read.c

```
@@ -0,0 +1,1082 @@
+        goto failed;
+    }
+
+    // Read file header blocks.
+    do {
+        num = readHeaderBlock(f, debugMode, fileName, &buf, &offset);
+    } while (num == 0);
+    if (num > 0) {
+        goto failed;
+    }
+
+    if (offset < dateEndPosition + 1) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)